plugin/bufferline.vim

```diff
@@ -106,13 +106,10 @@ function! bufferline#init_icons() abort
 
   " Default icons
   let g:bufferline.icons = extend({
```

Comment:
A few of these are unused, so I removed the ones which have no references in the codebase.